packages/office-ui-fabric-react/src/utilities/positioning/positioning.ts

```diff
@@ -417,12 +417,13 @@ export namespace positioningFunctions {
     elementRectangle: Rectangle,
     hostElement: HTMLElement,
     targetEdge: RectangleEdge,
-    alignmentEdge?: RectangleEdge
+    alignmentEdge?: RectangleEdge,
+    coverTarget?: boolean
```

Comment:
Just a quick comment to say that this addition makes it out of sync with the jsdoc... (I need to take a closer look when I have the time to see why a function like `_finalizePositionData` is being exported :))